submissions/m-ruslan/html-css-popup/index.html

```diff
@@ -0,0 +1,153 @@
+
+                <a href="#" class="app-cont" tabindex="1">
+                  <img src="img/contacts.png"  alt="Contacts icon">
+                  Contacts
+                </a>
+
+                <a href="#" class="app-cont" tabindex="1">
+                  <img src="img/tanslation.png"  alt="Translation icon">
+                  Translate
+                </a>
+
+                <a href="#" class="app-cont" tabindex="1">
+                  <img src="img/videos.png"  alt="Video icon">
+                  Videos
+                </a>
+
+                <a href="#" class="app-cont" tabindex="1">
+                  <img src="img/search.png"  alt="Search icon">
+                  Search
+                </a>
+
+                <a href="#" class="app-cont" tabindex="1">
+                  <img src="img/photos.png"  alt="Photos icon">
+                  Photo
+                 </a>
+
+                <a href="#" class="app-cont" tabindex="1">
+                  <img src="img/apps.png"  alt="Apps icon">
+                  Apps
+                </a>
+
+                <a href="#" class="app-cont" tabindex="1">
+                  <img src="img/documents.png"  alt="Documents icon">
+                  Document
+                </a>
+              </div>
+
+              <div id="apps-window2">
+                <div id="apps-block2">
+                  
+                  <a href="#" class="app-cont" tabindex="1"> 
+                    <img src="img/chat.png"  alt="Chat icon">
+                    Chat
+                  </a>
+
+                  <a href="#" class="app-cont" tabindex="1">
+                    <img src="img/classroom.png"  alt="Classroom icon">
+                    Classroom
+                  </a>
+
+                  <a href="#" class="app-cont" tabindex="1">
+                    <img src="img/collection.png"  alt="Collection icon">
+                    Collection
+                  </a>
+
+                  <a href="#" class="app-cont" tabindex="1">
+                    <img src="img/mars.png"  alt="Mars icon">
+                    Mars
+                  </a>
+
+                  <a href="#" class="app-cont" tabindex="1">
+                    <img src="img/blogs.png"  alt="Blogs icon">
+                    Blogs
+                  </a>
+
+                  <a href="#" class="app-cont" tabindex="1">
+                    <img src="img/ads.png"  alt="Ads icon">
+                    Promos
+                  </a>
+                  
+                </div>
+
+                <div id="even-more">
+                  <p>Even more from Kottans</p>
+                </div>
+                
+              </div>
+            </div>
+          </div>
+        </li>
+
+        <li class="nav-el">
+          <a href="#" tabindex="0"><img id="bell-icon" src="img/bell.png" alt="Notifications icon"></a>
+        </li>
+        <li class="nav-el">
+          <a href="#" tabindex="0"><img id="profile-photo" src="img/bilbo.jpeg" alt="profile-photo"></a>
```

Comment:
Try not to load too big images, because it has effect on loading speed.
 
<img width="985" alt="Screenshot 2020-10-26 at 12 29 18" src="https://user-images.githubusercontent.com/42999342/97164385-b5ebfe80-178a-11eb-9ee4-30040eb73c4e.png">

Reply:
Thank you, I've reduced the size of profile-photo image.
Also I've noticed that all other images are big sized too, but I havn't changed their size for now due to don't have any installed program that can reduce the size of png files, but I will take this into account in the future.